src/phoenix/db/migrations/versions/10460e46d750_datasets.py

```diff
@@ -0,0 +1,130 @@
+"""datasets
+
+Revision ID: 10460e46d750
+Revises: cf03bd6bae1d
+Create Date: 2024-05-10 11:24:23.985834
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from phoenix.db.migrations.types import JSON_
+
+# revision identifiers, used by Alembic.
+revision: str = "10460e46d750"
+down_revision: Union[str, None] = "cf03bd6bae1d"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "datasets",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("name", sa.String, nullable=False, unique=True),
+        sa.Column("description", sa.String, nullable=True),
+        sa.Column("metadata", JSON_, nullable=False),
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            nullable=False,
+            server_default=sa.func.now(),
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            nullable=False,
+            server_default=sa.func.now(),
+            onupdate=sa.func.now(),
+        ),
+    )
+    op.create_table(
+        "dataset_versions",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column(
+            "dataset_rowid",
```

Comment:
We could just do dataset_id since there is no collision

Reply:
addressed in d93886772eb537a739ecf212cd0da34cde420a9e